<launch>
  <node name="delta_lidar" pkg="delta_lidar" type="delta_lidar_node" output="screen" respawn="true">
    <param name="serial_port" type="string" value="/dev/lidar"/>
    <param name="frame_id" type="string" value="laser_frame"/>
    <param name="range_min" type="double" value="1.5"/>
    <param name="range_max" type="double" value="5.0"/>
  </node>
</launch>
